### PR TITLE
Fix: remove deprecated rand

### PR DESCRIPTION
**Pull request by fengshunli**
#### What this PR does / why we need it:

that need a specific result sequence should use NewRand(NewSource(seed))
image

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: cmd/yurt-controller-manager/controller-manager.go [hunk 1]
```diff
@@ -38,7 +38,8 @@ import (
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
+	newRand := rand.New(rand.NewSource(time.Now().UnixNano()))
+	newRand.Seed(time.Now().UnixNano())
```
Comment (rambohe-ch) [inline cmd/yurt-controller-manager/controller-manager.go:42]: would you be able to explain the reason of change rand seed?

Comment (fengshunli) [reply cmd/yurt-controller-manager/controller-manager.go:42]: see desc

Comment (rambohe-ch) [reply cmd/yurt-controller-manager/controller-manager.go:42]: good job.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**